pr2eus/robot-interface.l

```diff
@@ -1424,6 +1427,7 @@ Return value is a list of interpolatingp for all controllers, so (null (some #'i
          ;;
          (dotimes (i 2)
            (if (< (setq diff-len (norm (subseq (send diff :worldpos) 0 2))) 200) ;; move_base thre = 200mm
+             (if holonomic
```

Comment:
can we use `:go-pos-unsafe` instead of ':go-velocity` ? if this works, then it chance to remove this line and use `go-pos-unsafe` for both holonoic and non-holonomic robot.

Reply:
we can use `:go-pos-unsafe` for pr2 robot.
but for fetch, we need to tune `:rotation-gain` in `:go-pos-unsafe`
that is because of friction between wheel and the ground.
for rotating 45 degree, it is better to set `:rotation-gain` `2.0` for better motion.
however it depends on rotation angle and hardware situation.

Reference:
https://github.com/jsk-ros-pkg/jsk_robot/blob/master/jsk_fetch_robot/fetcheus/fetch-interface.l#L241-L323

Reply:
by the way, we checked gas spring of fetch yesterday with @708yamaguchi and @HiroIshida, and we need to replace it.